bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYZprime/proc_card_mg5.template

```diff
@@ -0,0 +1,9 @@
+import model_v4 $modelfile
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > zp ,  zp > ta- ta+  QCD=0
```

Comment:
Wouldn't it improve the modeling do have additional partons and use MLM?

Reply:
Maybe yes, perhaps just up to one additional parton. @florez any inputs from the point of view of the analysis selections?

Reply:
Adding +1 and +2j doesn't really change the signal acceptance for this analysis because we don't have any selections that create a significantly boosted topology. Please find a plot attached, where you can see that the largest change in acceptance is < 3.5% for low Z' masses. In addition, our selection criteria does not relay  on MET, neither our main observable to search for signal or establish limits. We use the reconstructed mass between the two taus and Delta_pT (vectorial difference in pT between the two associated taus from the Z'). Therefore, our analysis is not sensitive to changes in MET resolution from ISR activity. We would like to maintain our LO production. We will calculate the corresponding k-factor for NLO processes and re-weight our cross sections accordingly.
[DYZprimeSignaAcceptance_Z0j_vs_Zjets.pdf](https://github.com/cms-sw/genproductions/files/5945601/DYZprimeSignaAcceptance_Z0j_vs_Zjets.pdf)